app/forms/submissions/trial/step04_content_form.rb

```diff
@@ -1,15 +1,25 @@
 module Submissions
   module Trial
     class Step04ContentForm < PlantTrialForm
+      property :trait_mapping, writeable: false
       property :trait_scores, writeable: false
+      property :accessions, writeable: false
+      property :lines_or_varieties, writeable: false
+      property :replicate_numbers, writeable: false
+      property :design_factor_names, writeable: false
+      property :design_factors, writeable: false
       property :upload_id
-      property :trait_mapping, writeable: false
 
       def self.permitted_properties
```

Comment:
No, all properties are permitted by default: https://github.com/eSpectrum-IT/brassica/blob/master/app/forms/base_form.rb#L5